commons/helpers/pods_helper.py

```diff
@@ -581,18 +581,22 @@ def apply_k8s_deployment(self, file_path: str):
                                 read_lines=True,exc=False)
         return True, resp
 
-    def select_random_pod_container(self,pod_prefix: str,
-                                    container_prefix: str):
+    def select_random_pod_container(self, pod_prefix: str,
+                                    container_prefix: str, **kwargs):
         """
         Select random pod and container for the given pods and container prefix
-        :param master_node: Logical Node object for master node.
         :param pod_prefix: Pod prefix
```

Comment:
Maybe we need to add to docstring that pod_prefix can be prefix or name in case of specific pod

Reply:
fixed.